src/DropzoneArea.js

```diff
@@ -304,4 +308,8 @@ DropzoneArea.propTypes = {
     onDelete: PropTypes.func
 };
 
+DropzoneArea.defaultProps = {
```

Comment:
Ahh, I think this is a little code is a little bit behind the current code now, requires the latest changes from master